### PR TITLE
add community page link to etcd.cn

### LINKED ISSUE #357: Community: add link to etcd.cn
Add a link to etcd.cn to the Community page, with a disclaimer that the site isn't officially endorsed by the etcd.io maintainers.

### PR DESCRIPTION
**Pull request by kapunahelewong**
Adds a section to content/en/community/_index.md with a link to etcd.cn 

Fixes #357

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-359--etcd.netlify.app/community/#additional-community-resources

**Review by nate-double-u — APPROVED** [review]
Thanks for this @kapunahelewong!

Looks good to me.

File: content/en/community/_index.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ submitting a pull request.
 
 For etcd contribution guidelines, see [How to contribute][].
 
+## Additional community resources
+
+Our dedicated community has created the following resource:
+
+* [Chinese version of this site](https://etcd.cn) (此网站的中文版). Note that the Chinese language site is an independent project and isn't officially endorsed by the etcd.io maintainers.
```
Comment (chalin) [inline content/en/community/_index.md:93]: A few tweaks:

- "version of this site" --> "translation of etcd.io"
- Put "(此...版)" inside / as part of the link text. Actually, use this text: "(本网站中文翻译)"
- ". Note that the Chinese language site is ..." --> "-- the website is ..."
- "by the etcd.io" --> "by etcd"

**Review by chalin — CHANGES_REQUESTED** [review]
Nice! See inlined comment for a few tweaks.

**Comment by chalin** [comment]
@nate-double-u - you were quicker than I was. Can either of you address the comments given in https://github.com/etcd-io/website/pull/359#discussion_r645120564? Thanks.

**Pull request merged**